meshnets/utils/data_visualization.py

```diff
@@ -107,5 +107,35 @@ def plot_mesh_comparison(mesh_path: str,
                      cmap='RdBu_r',
                      clim=clim,
                      copy_mesh=True)
+    plotter.show(screenshot=screenshot)
+
+
+def plot_relative_error(mesh_path: str,
+                        ground_truth: np.array,
+                        prediction: np.array,
+                        clim: Union[Tuple[float, float], None] = None,
+                        rot_z: int = 0,
+                        off_screen: bool = False,
+                        screenshot: Union[str, None] = None):
```

Comment:
Could you add here a description of what you meant in the PR to change the color bar.

Reply:
Basically there are outliers in the predictions that have very large errors. Therefore, we we use the colorbar with range [0, max_error] it becomes very hard to differentiate between the smaller errors as they will be assigned indistinguishable colors. Thus, I make a second plot with the colorbar values clipped to the range [0, 1] so that we can take a look at the smaller errors

Reply:
But I thought that this will always be less 1, since it is the relative error.